Rovers/Rovers.tex

```diff
@@ -0,0 +1,344 @@
+The competition is held annually and features four very intense competitions. 
+Such a rover would need to be fully self navigating, perform scientific analysis on soil samples, and having a robotic arm capable of fine motor control. 
+Such a project would be among the most ambitious SPEX has ever attempted. 
+These features line up with the improvements section of the MVP.
+The skills learned from this project should be applicable to such a future project. 
+This secondary objective is directly related to the long-term vision of the rover project. 
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+
+\subsection{Space Exploration}
+Rovers are a huge part of space exploration. 
+There are eight active missions on Mars. 
+Four of those include rovers. 
+To ignore this fact would be to ignore a large part of space exploration.
+Currently RIT SPEX is not involved with rovers.
+It would be beneficial to our members to get some experience in this area.
+
+\subsection{Promotions and Funding}
+A rover would look super good for SPEX at Imagine RIT. 
+The rover would be rather large and would attract many eyes. 
+We could even demo it outside if there is sufficient space. 
+A rover would be very easy to get video and photos of for SPEX promotions. 
+Having a rover is also another opportunity for SPEX to fundraise. 
+There is plenty of space to place company logos on the body of the rover. 
+It would also allow for SPEX to reach out to robotics companies.
+
+\subsection{Computer Science}
+\subsubsection{Member Retention and Expansion}
+The potential self-driving component would be the heaviest computer science project SPEX would have attempted, this would help with retention of CS and SE majors. 
+There are many features included in the MVP that require significant computer science skills as well.
+This includes navigation, power management, communications and more.
+That will give computing-related majors another option for SPEX involvement, also helping retention.
+
+\subsubsection{Artificial Intelligence}
+Some of the potential self-driving features will likely implemented with artificial intelligence.
+Machine learning and artificial intelligence are at the forefront of computer science right now. 
+They are heavily desired in industry, including space exploration.
+There are limited options for students to get involved with AI project wise, this can help with that. 
+
+\subsection{Robotics}
+Robotics are also a heavily desired area in the space industry. 
+Currently RIT SPEX does not have any projects with robotics as a focus. Robotics is also a potential area for collaboration with faculty as there are RIT faculty that are investigating this area.
+RIT has limited options for students to get involved with robotics just as it has limited options related to space exploration. 
+
+\subsection{Challenge}
+The project would be very ambitious. 
+It would require at a minimum a handful (4-5) members each with different areas of expertise. 
```

Comment:
Robotics, mechanisms, electronics, software, and AI are the disciplines outlined above. Assuming team members will specialize in one discipline (obviously everyone will probably work on everything, but keyword is specialize here) a team of 4-5 students leaves one person for each discipline. If this number is the minimum, what is the ideal number of students on the team?

Reply:
Ideally for the MVP 7 members. For a good mix of the improvements 15-20